src/carnot/funcs/builtins/pprof.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <sys/sysinfo.h>
+
+#include <absl/container/flat_hash_map.h>
+
+#include <string>
+
+#include "src/shared/pprof/pprof.h"
+
+#include "src/carnot/udf/registry.h"
+#include "src/carnot/udf/udf.h"
+
+namespace px {
+namespace carnot {
+namespace builtins {
+
+using px::shared::PProfProfile;
+
+class CreatePProfRowAggregate : public udf::UDA {
+ public:
+  static udf::UDADocBuilder Doc() {
+    return udf::UDADocBuilder("Convert perf profiling data to pprof format.")
+        .Details("Converts perf profiling stack traces into pprof format.")
+        .Example("df = df.agg(pprof=('stack_trace', 'count', px.pprof))")
```

Comment:
I think it would be helpful to provide a slightly more complete example (referencing the `stack_traces.beta` data table).

As an aside, do we have plans to rename that table? It wasn't until I looked up the name of it that I realized it was suffixed with "beta".

Reply:
Expanded the example and opened issue #1351 to move the perf profiler table out of beta.